algorithms/gr-dij/csharp/DijkstraAlgorithm.cs

```diff
@@ -0,0 +1,92 @@
+// A C# program for Dijkstra's  shortest path algorithm. 
+using System; 
+
+class Dijkstra { 
+
+	static int V = 9; 
+
+	static int minDistance(int[] dist, bool[] sptSet) 
+	{ 
+		// Initialize min value 
+		int min = int.MaxValue, min_index = -1; 
+
+		for (int v = 0; v < V; v++) 
+		{
+			if (sptSet[v] == false && dist[v] <= min) 
+			{ 
+				min = dist[v]; 
+				min_index = v; 
+			} 
+		}
+
+		return min_index; 
+	} 
+
+	// Function to print the constructed distance array 
+	static void printSolution(int[] dist) 
+	{ 
+		Console.Write("Vertex \t\t Distance "
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6349348)